packages/lab/src/carousel/CarouselSlide.tsx

```diff
@@ -1,76 +1,134 @@
-import { makePrefixer } from "@salt-ds/core";
+import { makePrefixer, useForkRef } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import { clsx } from "clsx";
 import {
-  type ElementType,
   type HTMLAttributes,
-  type ReactElement,
+  type ReactNode,
   forwardRef,
+  useEffect,
   useRef,
+  useState,
 } from "react";
-
-import { clsx } from "clsx";
+import { useCarousel } from "./CarouselContext";
+import carouselSlideCss from "./CarouselSlide.css";
 
 export interface CarouselSlideProps extends HTMLAttributes<HTMLDivElement> {
-  ButtonBar?: ElementType;
-  Media: ReactElement;
-  description?: string;
-  title?: string;
-  contentAlignment?: "center" | "left" | "right";
+  /**
+   * Actions to be displayed in the content footer.
+   */
+  actions?: ReactNode;
```

Comment:
A Carousel's responsibility is to just navigate content, it does not define what the content is.

Actions on the API , this forces a single pattern on every slide.
Ideally you can just compose Salt Cards in the Carousel and then this is a Cards pattern ?

Maybe this extends to media?

Reply:
if the content has navigable items and we dont ensure those are hidden then tab would move slides